include/micm/solver/rosenbrock.hpp

```diff
@@ -188,7 +188,7 @@ namespace micm
     /// @param rate_constants List of rate constants for each needed species
     /// @param number_densities The number density of each species
     /// @param jacobian The matrix of partial derivatives
```

Comment:
```suggestion
    /// @param jacobian The matrix of negative partial derivatives
```
Maybe also update the brief description of this function above to indicate that it returns the negative of the Jacobian

Reply:
Done.